api/helpers/gamestate/deal-cards.js

```diff
@@ -0,0 +1,80 @@
+const GameStatus = require('../../../utils/GameStatus.json');
+const DeckIds = require('../../../utils/DeckIds.json');
+const MoveType = require('../../../utils/MoveType.json');
+const GamePhase = require('../../../utils/GamePhase.json');
+
+module.exports = {
+  friendlyName: 'Deal Cards',
+
+  description: 'Creates initial GameStateRow for specified game, with cards dealt ot both players (5 for p0 and 6 for p1). Errors if Game has previous game or incorrec status.',
+
+  inputs: {
+    game: {
+      type: 'ref',
+      description: 'Game record to be initialized with first GameState',
+      extendedDescription: 'Game is expected to have populated collection of .gameStates',
+      required: true,
+    },
+  },
+
+  fn: async ({ game }, exits) => {
+    
+    try {
+
+      if (game.status !== GameStatus.STARTED) {
+        return exits.error({ message: 'Game has not yet started or is over' });
```

Comment:
Todo: check the UX of these errors. They shouldn't generally occur through the client, but if they do, a snackbar would be appropriate

Reply:
I've updated the errors so they flow better through the endpoints, but the lobby doesn't have an error snackbar. Let's write a separate issue to add one